package.json

```diff
@@ -96,6 +96,7 @@
     "js-string-escape": "^1.0.0",
     "loader.js": "^4.7.0",
     "mkdirp": "^0.5.1",
+    "pretender": "^3.0.4",
```

Comment:
Do we need to switch any imports with the switch from ember-cli-pretender to pretender?

Reply:
No, ember-cli-pretender made `import Pretender from 'pretender'` work

Reply:
We are already importing from pretender: https://github.com/emberjs/ember-inspector/blob/master/tests/acceptance/whats-new-test.js#L4